lib/auth/auth.ts

```diff
@@ -121,7 +121,8 @@ function doAuth(
     log: Logger,
     cb: (err: Error | null, data?: any) => void,
     awsService: string,
-    requestContexts: any[] | null
+    requestContexts: any[] | null,
+    options: any = {},
```

Comment:
same as https://github.com/scality/Arsenal/pull/2288/files#r1977465725

Reply:
@williamlardier Will #2299 handle typing of `options` as well, or just `requestContext`? There will be conflicts anyway, but is it better to keep `options` untyped here, and type it in the next PR, as it will make the work smoother?

(we probably want to merge this PR first, to unblock Utapi?)

Reply:
Ye, I think it's better not to type it. I will ping this PR if my PR is merged before, but this is not the logical order indeed. I can handle it in #2299.